src/test/regress/expected/upgrade_distributed_triggers_before.out

```diff
@@ -241,3 +241,6 @@ SELECT * FROM sale_triggers ORDER BY 1, 2;
  truncate_trigger_xxxxxxx      | sale_california | O
 (12 rows)
 
+-- check that we can't rename child triggers on partitions of distributed tables
+ALTER TRIGGER another_trigger ON sale_newyork RENAME TO another_renamed_trigger;
+ERROR:  cannot rename child triggers on distributed partitions
```

Comment:
This error message will make it harder for us to add PG16 support in the future. As this error happens on versions before PG15, this test will fail on PG15->PG16 upgrades.

How about adding the test in the regression suite in a new test file that will be run only for PG versions earlier than 15?

Reply:
Actually this test is only run in PG14 so we're good.